src/realtimecontroller/DefaultRealtimeController.ts

```diff
@@ -317,6 +317,12 @@ export default class DefaultRealtimeController implements RealtimeController {
     });
   }
 
+  realtimeExternalUserIdFromAttendeeId(attendeeId: string): string | null {
+    if (attendeeId in this.state.attendeeIdToExternalUserId) {
```

Comment:
What does this function return if `attendeeId` is not `attendeeIdToExternalUserId`?